src/neo4j_genai/kg_construction/types.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from __future__ import annotations
+
+from typing import Any, Optional
+
+from pydantic import BaseModel
+
+
+class Neo4jProperty(BaseModel):
+    key: str
+    value: Any
+
+
+class Neo4jEmbeddingProperty(BaseModel):
+    key: str
+    value: str
```

Comment:
Shouldn't the type of value be `list[float]`?

Reply:
Oh ok, I see. It makes sense to me to delegate the embedding creation to another component. This way we can probably use the same component to embed the chunks if we want to.

Reply:
Have removed it. Will raise a new PR with the embedding component.